File: 2023_1/XDES01/Aula4/ex10.c
#include <stdio.h>

int main() {
    int piecesAmount = 0, hours = 0, minutes = 0, seconds = 0, totalSeconds = 0;

    scanf("%d", &piecesAmount);

    totalSeconds = piecesAmount * 17;

    seconds = totalSeconds % 60;
    minutes = (totalSeconds / 60) % 60;
    hours = (totalSeconds / (60 * 60)) % 60;

    printf("%d %d %d\n", hours, minutes, seconds);

    return 0;
}

File: 2024_2/XDES02/Aula01/ex10.py
from datetime import datetime, timedelta

def timeToSeconds(timeStr: str):
	timeStamp = datetime.strptime(timeStr, '%H:%M:%S')
	totalSeconds = (timeStamp.hour * 60 * 60) + (timeStamp.minute * 60) + timeStamp.second

	return totalSeconds


def secondsToTime(totalSeconds: int):
	seconds = totalSeconds % 60
	minutes = int(((totalSeconds - seconds) / 60) % 60)
	hours = int((totalSeconds - (minutes * 60) - seconds) / (60 * 60))

	return f"{hours:02}:{minutes:02}:{seconds:02}"

checkInTime = input("Check-in time (HH:MM:ss): ")
checkOutTime = input("Check-out time (HH:MM:ss): ")

checkInSeconds = timeToSeconds(checkInTime)
checkOutSeconds = timeToSeconds(checkOutTime)

elapsedTime = secondsToTime(checkOutSeconds - checkInSeconds)

print(f"Elapsed time was: {elapsedTime}")
